css/flashcards.css

```diff
@@ -529,12 +529,14 @@
 }
 
 .h5p-flashcards .h5p-solution > span > span {
-  font-size: 1.5em;
+  font-size: 1.25em;
+  line-height: 1.25em;
   vertical-align: bottom;
 }
 
 .h5p-flashcards .h5p-solution > span > span > span{
   font-size: initial;
+  margin-left: -0.2em;
```

Comment:
We should really avoid selectors like span > span >span, since it will be very hard later to understand which element this is hitting. In addition, performance wise we give the browser a harder job. A much better solution is to give it a class name, i.e h5p-flashcards-solution-separator.